test/jdk/java/lang/String/TranslateEscapes.java

```diff
@@ -86,14 +87,53 @@ static void test4() {
         verifyLineTerminator("\r");
     }
 
+    /*
+     * Unicode escapes.
+     */
+    static void test5() {
+        verifyUnicodeEscape("\\u0000", "\u0000");
+        verifyUnicodeEscape("\\u2022", "\u2022");
+        verifyUnicodeEscape("\\ud83c\\udf09", "\ud83c\udf09");
+        verifyUnicodeEscape("\\uuuuu2022", "\uuuuu2022");
```

Comment:
Include the code from the example as a test case too.

Reply:
None present. Was a mis-paste.